src/python/utils.hpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <pybind11/pybind11.h>
+#include <pybind11/embed.h>
+#include <pybind11/stl.h>
+namespace py = pybind11;
+using namespace py::literals;
+
+namespace ovms {
+
+template <class T = py::object>
```

Comment:
If T can be **only** py::object, why do we make it templated? Can't we just swap T with py::object?

Reply:
T can be any type of object, but if unspecified it's py::object (as a default)

Reply:
Do we need that ambiguity?